fastlane_bot/data/abi.py

```diff
@@ -155,6 +155,20 @@
         "stateMutability": "view",
         "inputs": [],
         "outputs": [{"internalType": "uint32", "name": "", "type": "uint32"}]
+    },
+    {
+        "type": "function",
+        "name": "deleteStrategy",
+        "stateMutability": "nonpayable",
+        "inputs": [{"internalType": "uint256","name": "strategyId","type": "uint256"}],
+        "outputs": []
+    },
+    {
+        "type":"function",
+        "name":"createStrategy",
+        "stateMutability":"payable",
+        "inputs":[{"internalType":"Token","name":"token0","type":"address"},{"internalType":"Token","name":"token1","type":"address"},{"components":[{"internalType":"uint128","name":"y","type":"uint128"},{"internalType":"uint128","name":"z","type":"uint128"},{"internalType":"uint64","name":"A","type":"uint64"},{"internalType":"uint64","name":"B","type":"uint64"}],"internalType":"structOrder[2]","name":"orders","type":"tuple[2]"}],
+        "outputs":[{"internalType":"uint256","name":"","type":"uint256"}]
```

Comment:
These are write functions (changing contract state).
They are not supposed to be used by the bot, even though they can be executed in read-only mode (aka *static call*).
If this is indeed how you're using them, then there is very likely a better way (i.e., calling an actual read function in the same contract).